schema.sql

```diff
@@ -1,12 +1,29 @@
 /* Database schema to keep the structure of entire database. */
-DROP table IF EXISTS animals;
+
+DROP TABLE IF EXISTS animals;
+DROP TABLE IF EXISTS owners;
+DROP TABLE IF EXISTS species;
+
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name varchar(100),
+    AGE int
+);
+
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name varchar(100)
+);
 
 CREATE TABLE animals (
-    id serial primary key,
+    id SERIAL PRIMARY KEY,
     name varchar(100),
     date_of_birth date,
     escape_attempts int,
     neutered boolean,
     weight_kg decimal,
-    species varchar(100)
-);
+    species_id INT,
+    owner_id INT, 
+    CONSTRAINT fk_species FOREIGN KEY(species_id) REFERENCES species(id),
+    CONSTRAINT fk_owners FOREIGN KEY(owner_id) REFERENCES owners(id)
+);
```

Comment:
- [ ] We are not supposed to edit this codes, you were supposed to write new instructions to change these field as you needed, example: `ALTER TABLE table_name ADD column_name data_type REFERENCES parent_table(primary_key_from_parent_table);`

Reply:
Thank you for your feedback @descholar-ceo . 
I will fix this right away.

Reply:
@Whoistolu I don't see the changes requested by @descholar-ceo. May you please confirm?

Reply:
Hello @oxenprogrammer I haven't implemented the required changes recommended. Kindly, help invalidate the second submission so I can implement the changes by using ALTER for the TABLE instead of DROP. Thanks in anticipation.

Reply:
I have just fixed the issue using `ALTER` command to modify the animals `TABLE` instead of `DROP` command.Issue now fixed using `ALTER` command to modify the animals `TABLE` instead of `DROP` command.